src/components/App/App.tsx

```diff
@@ -46,6 +46,9 @@ const VoucherRedemptionPage = lazy(
 const MerchantVoucherDashboard = lazy(
   () => import('../../pages/MerchantVoucherDashboard')
 );
+const MerchantVoucherDashboardV2 = lazy(
```

Comment:
I think this (and the other merchant dashboard stuff) might be a merge conflict with @chrisng93's work -- the V2 has been removed and the regular one updated. Would you mind re-resolving this?